sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -831,7 +839,7 @@ def expand(self, pcoll):
       self.table_reference.projectId = pcoll.pipeline.options.view_as(
           GoogleCloudOptions).project
 
-    if standard_options.streaming:
+    if standard_options.streaming or self.method == 'STREAMING_INSERTS':
```

Comment:
Should print a warning or fail if a user is explicitly setting FILE_LOADS in a Batch pipeline?

Reply:
Added.